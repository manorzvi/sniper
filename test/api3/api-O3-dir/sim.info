{'cmdline': ['../../run-sniper',
             '-n',
             '1',
             '-d',
             'api-O3-dir',
             '--',
             './api-O3'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (6.667772,
            0.072651,
            38328,
            0,
            0,
            0,
            8028,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            20734,
            6),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api3/api-O3-dir --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpV2vyzy/run_benchmarks'],
 't_elapsed': 10.602528095245361,
 't_start': 1633352618.05022,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpV2vyzy/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api-O3']],
 'user': 'root',
 'vmem': 220901376L}
