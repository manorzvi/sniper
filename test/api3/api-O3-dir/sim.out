                                     | Core 0    
  Instructions                       |   12621632
  Cycles                             |    4486888
  IPC                                |       2.81
  Time (ns)                          |    1686800
  Idle time (ns)                     |         14
  Idle time (%)                      |       0.0%
Branch predictor stats               |           
  num correct                        |    2522008
  num incorrect                      |       1517
  misprediction rate                 |      0.06%
  mpki                               |       0.12
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |      15340
    num misses                       |         58
    miss rate                        |      0.38%
    mpki                             |       0.00
  D-TLB                              |           
    num accesses                     |      40252
    num misses                       |         95
    miss rate                        |      0.24%
    mpki                             |       0.01
  L2 TLB                             |           
    num accesses                     |        153
    num misses                       |        127
    miss rate                        |     83.01%
    mpki                             |       0.01
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |      15340
    num cache misses                 |        895
    miss rate                        |      5.83%
    mpki                             |       0.07
  Cache L1-D                         |           
    num cache accesses               |      40252
    num cache misses                 |       1804
    miss rate                        |      4.48%
    mpki                             |       0.14
  Cache L2                           |           
    num cache accesses               |       2737
    num cache misses                 |       2492
    miss rate                        |     91.05%
    mpki                             |       0.20
  Cache L3                           |           
    num cache accesses               |       2532
    num cache misses                 |       2481
    miss rate                        |     97.99%
    mpki                             |       0.20
DRAM summary                         |           
  num dram accesses                  |       2481
  average dram access latency (ns)   |      73.04
  average dram queueing delay        |      19.61
  average dram bandwidth utilization |      1.00%
Coherency Traffic                    |           
  num loads from dram                |       1117
  num loads from dram cache          |          0
  num loads from remote cache        |          0
